app/src/main/res/layout/activity_dashboard.xml

```diff
@@ -203,6 +203,10 @@
 
                 </androidx.constraintlayout.widget.ConstraintLayout>
 
+                <include
+                    android:id="@+id/country_info"
+                    layout="@layout/partial_country_info" />
+
```

Comment:
We will transform this into view holder component later

Reply:
yes, maybe we can transform it in your PR, I just made it as partial layout to make it easier to refactor

Reply:
Roger!